common/lib/common-definitions/src/fluidError.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryProperties } from "./logger";
```

Comment:
I'm not a big fan of putting code in into common-utils.
Why not put it into telemetry-utils package? Or have it in its own package?
This is client-only code, common-utils is for code that is shared between client & server (at least right now).

Reply:
Need it here to support `assert` in both client and server code.  e.g. there are some specific asserts in quorum that we want to mark as `dataIntegrityError`.